Cinema system/Cinema system/Services/CinemaService.cs

```diff
@@ -0,0 +1,151 @@
+﻿using CinemaSystem.Interfaces;
+using CinemaSystem.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System.ComponentModel.DataAnnotations;
+
+namespace CinemaSystem.Services
+{
+    public class CinemaService : ICinemaService
+    {
+
+        private readonly CinemaDBContext _db;
+
+        public CinemaService(CinemaDBContext db)
+        {
+            _db = db;
+        }
+
+        public Errors IsCinemaExists(int cinemaId)
+        {
+            Errors errors = new Errors();
+
+            if (!_db.Cinemas.Any(c => (c.Id == cinemaId && !c.IsDeleted)))
+            {
+                errors.IsCorrect = false;
+                errors.Description.Add("This cinema does not exists.");
+                return errors;
+            }
+
+            errors.IsCorrect = true;
+            return errors;
+        }
+
+        public Errors IsCinemaExistsAndValid(CinemaView cinema)
```

Comment:
If you want to use 'And' in method name this is the sign that you should have two separate methods instead.